### DISCUSSION TITLE
Contact data for a group

### DISCUSSION THREAD

**Discussion by freddepettersson** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#70] [Q&A] [answered]
What do you think about adding contact data to a group ?
Type : Email and website  only

**Comment by fjallemark** [comment, marked as answer]
The definition is *A group is any set of people that has a reason to administer modules collectively.* This may or many not coincident with actual organisations. I am also restrictive about how much information should be in the Module Registry. The primary purpose is to handle data about modules and meetings. Contact mediation between individuals and groups, similar web links and public email addresses have so far not been on the agenda. This places additional demands on the handling of personal data, and that the contact information is kept up to date. For example, no contact information on the contact page is provided. Instead, you are asked to use existing channels to get in touch. This is intentional in order not to expose more data than absolutely necessary. So adding website and contact email has to wait until there is a clear purpose and use for it with in the module community.
